app/controllers/validates_controller.rb

```diff
@@ -9,7 +9,7 @@ def show
       work = Work.find(params[:work_id])
       work_version = work.head
     else
-      work = Work.new(collection_id: params[:collection_id])
+      work = Work.new(collection_id: params[:collection_id], depositor: User.new)
```

Comment:
the new `agree_to_terms` assumes there is an associated depositor, we need one for a new work